analysis/genie/variant_analysis/genie_search_analysis.ipynb

```diff
@@ -1239,32 +1265,36 @@
     "    :param n, d: Two integers\n",
     "    :return: A string reporting the percentage\"\"\"\n",
     "    percent = round(100 * n / d, 1)\n",
-    "    return f\"{n} / {d} ({percent}%)\"\n",
+    "    return percent\n",
```

Comment:
You need to update the return type and docstring. For params, you create two separate lines: 

```python
:param param1:
:param param2:
```